### PR TITLE
email template added

### PR DESCRIPTION
**Pull request by sabrina34352**
image

### CODE REVIEW

File: packages/email-service/template/index.hbs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            height: 37px;
+            margin: auto;
+            color: white;
+            font-weight: 700;
+            font-size: 14px;
+            line-height: 17px;
+            border:none;
+        }
+        .mimir{
+            font-family: 'Montserrat';
```
Comment (unrealsolver) [inline packages/email-service/template/index.hbs:86]: Please, use `ui-kit` library and insert these design items as parameters.

File: packages/email-service/template/index.hbs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            </p>
+          </div>
+        {{/each}}
+        <button class='button'>Go to app</button>
+        <p class='mimir'>MIMIR</p>
+      </div>
+    </div>
+  </body>
+
+</html>
```
Comment (unrealsolver) [inline packages/email-service/template/index.hbs:116]: Add `LF` at the `EoL`

**Review by unrealsolver — CHANGES_REQUESTED** [review]
Please convert png to jpg. 700KB are too much for an email.

File: packages/email-service/template/index.hbs (hunk 1 above)
Comment (sabrina34352) [reply packages/email-service/template/index.hbs]: its not allowing it, even `background-image` property is not working

**Pull request merged**